src/main/java/net/sf/jabref/logic/importer/fetcher/AbstractIsbnFetcher.java

```diff
@@ -0,0 +1,37 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import net.sf.jabref.logic.help.HelpFile;
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.logic.importer.IdBasedParserFetcher;
+import net.sf.jabref.logic.importer.ImportFormatPreferences;
+import net.sf.jabref.logic.importer.Parser;
+import net.sf.jabref.logic.importer.fileformat.BibtexParser;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.util.ISBN;
+
+public abstract class AbstractIsbnFetcher implements IdBasedParserFetcher {
```

Comment:
In IsbnViaChimboriFetcher, I need both `doPostCleanup()` and `getParser()` which are not offered by `IdBasedFetcher`.

Don't want to introduce a new intermediate class...